gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 format.version = "1.1"
 
 [versions]
-edc = "0.0.1-20230426-SNAPSHOT"
+edc = "0.0.1-SNAPSHOT"
```

Comment:
`0.0.1-20230506-SNAPSHOT` has been published

Reply:
I will change that on nightly before merging to the main branch. For now in this branch we need to track the latest changes